AppNavBar - error TS2304: Cannot find name 'mixed'. Typescript error.

<!--- Provide a general summary of the issue in the Title above -->
<!--- Provide a sandbox that reproduces your issue - you can fork one from  https://baseweb.design/ -->

## Current Behavior
Having this error on `yarn build` or `yarn tsc`
```
❯ yarn tsc
yarn run v1.22.10
warning package.json: No license field
$ /node_modules/.bin/tsc
node_modules/baseui/app-nav-bar/index.d.ts:21:42 - error TS2304: Cannot find name 'mixed'.

21   onUserItemSelect?: (item: NavItemT) => mixed;
                                            ~~~~~

node_modules/baseui/app-nav-bar/index.d.ts:28:42 - error TS2304: Cannot find name 'mixed'.

28   onMainItemSelect?: (item: NavItemT) => mixed;
                                            ~~~~~


Found 2 errors.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
 
```
## Expected Behavior

No error. Smooth compile.

## Your Environment

<!--- Include as many relevant details about the environment with which you experienced the bug. -->

| Tech    | Version |
| ------- | ------- |
| Base UI | ^9.112.0  |
| React   |   ^17.0.0      |
| browser |         |

- [x] I have searched the [issues](https://github.com/uber/baseweb/issues) of this repository and believe that this is not a duplicate.

--- a/src/app-nav-bar/index.d.ts
+++ b/src/app-nav-bar/index.d.ts
@@ -18,14 +18,14 @@ export type UserMenuPropsT = {
   username?: string;
   usernameSubtitle?: React.ReactNode;
   userImgUrl?: string;
-  onUserItemSelect?: (item: NavItemT) => mixed;
+  onUserItemSelect?: (item: NavItemT) => any;
 };
 
 export type AppNavBarPropsT = UserMenuPropsT & {
   isMainItemActive?: (item: NavItemT) => boolean;
   mainItems?: NavItemT[];
   mapItemToNode?: (item: NavItemT) => React.ReactNode;
-  onMainItemSelect?: (item: NavItemT) => mixed;
+  onMainItemSelect?: (item: NavItemT) => any;
   title?: React.ReactNode;
 };
 

--- a/src/tooltip/__tests__/tooltip-interactive-element.scenario.js
+++ b/src/tooltip/__tests__/tooltip-interactive-element.scenario.js
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2018-2020 Uber Technologies, Inc.
+Copyright (c) Uber Technologies, Inc.
 
 This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.

